service/controller/machinepoolexp/handler/migration/testdata/old_machinepool.yaml

```diff
@@ -0,0 +1,95 @@
+apiVersion: exp.cluster.x-k8s.io/v1alpha3
```

Comment:
Test input, old experimental MachinePool that will be migrated to the new non-experimental MachinePool.